model/registry/db.yaml

```diff
@@ -161,7 +168,8 @@ groups:
               stability: experimental
             - id: firstsql
               value: 'firstsql'
-              brief: 'FirstSQL'
+              deprecated: "Removed."
+              brief: "Deprecated, no replacement at this time."
```

Comment:
Replaced by `other_sql`? (with our internal reasoning being that we don't need to list _every_ database that ever existed)